lib/Parser/EscriptGrammar/EscriptParser.g4

```diff
@@ -44,9 +44,30 @@ topLevelDeclaration
     | includeDeclaration
     | programDeclaration
     | functionDeclaration
+    | classDeclaration
     | statement
     ;
 
+classDeclaration
+    : CLASS IDENTIFIER classParameters? classBody ENDCLASS
```

Comment:
In next pr: make class parameters required, as we don't have functions without functionParameters

Reply:
Not next pr, but in followup grammar changes PR, as we also need to support `::identifier`